demo/assets/hand_outline.gdshader

```diff
@@ -0,0 +1,20 @@
+shader_type spatial;
+
+render_mode cull_front,unshaded;
+uniform vec3 outline_color:source_color = vec3(0.26,0.34,0.40);
```

Comment:
We usually use more whitespace than this, for example:

```suggestion
render_mode cull_front, unshaded;
uniform vec3 outline_color : source_color = vec3(0.26, 0.34, 0.40);
```

However, I don't think that's super important or should necessarily hold up this PR.